api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortQueryBuilder.java

```diff
@@ -98,7 +98,11 @@ public QueryJobConfiguration buildUnionedParticipantIdQuery(
         .build();
   }
 
-  public QueryJobConfiguration buildPersonIdQuery(String personIdQuery) {
-    return QueryJobConfiguration.newBuilder(personIdQuery).setUseLegacySql(false).build();
+  public QueryJobConfiguration buildWGSPersonIdQuery(String personIdQuery) {
```

Comment:
since this appears to be Tanagra-specific, what do you think about reflecting that in the name or adding a comment?